### PR TITLE
Add in an app usage page (bug 1129065)

### PR DESCRIPTION
**Pull request by andymckay**
apologies for the noise, moved over to a pull request from my repo instead of one on the master

### CODE REVIEW

**Comment by andymckay** [comment]
r? @diox and @ngokevin 

new pull request, 2nd commit has all the feedback @ngokevin provided in the first pull request.

**Comment by andymckay** [comment]
Old pull request, for reference is here: https://github.com/mozilla/fireplace/pull/1022/files#r24706422

File: src/templates/usage.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<section class="main">
+  <div class="subheader">
+    <h1>
+      {{ _('App Statistics') }}
```
Comment (ngokevin) [inline src/templates/usage.html]: Which app is this for? Marketplace?

Comment (andymckay) [inline src/templates/usage.html]: Bug 1129065, see https://wiki.mozilla.org/Marketplace/AppStats

**Comment by diox** [comment]
Not wild about standalone.styl.css name either but I don't have any alternate suggestion, so r+

**Pull request merged**